src/Sylius/Bundle/ApiBundle/Validator/Constraints/OrderShippingMethodAvailable.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class OrderShippingMethodAvailable extends Constraint
```

Comment:
```suggestion
final class OrderShippingMethodAvailability extends Constraint
```

Reply:
I moved the logic to the Eligibility validator.